### DISCUSSION TITLE
Testing file filter rules

### DISCUSSION THREAD

**Discussion by posixltd** [WinMerge/winmerge#591] [Q&A] [answered]
When Testing file filter rules (manual section 4.1.4), does result "Failed" mean the filter is syntactically incorrect, or that the filter has failed to find something to filter, or something else?

**Comment by sdottaka** [comment, marked as answer]
The result "Failed" should indicate that the specified file or folder name is treated as an item that was ignored when comparing folders.

**Reply by posixltd** [reply]
Thank you!  May I suggest you add your response to the manual?  It might help others later.  Cheers, Richard
